programmer-humour/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adrian Ilovan</title>
+    <style>
+        #image {
+            max-width: 100%;
+            height: auto;
+        }
+    </style>
+</head>
+<body>
+    <h1>Image API</h1>
+    <div class="container">
+        <img src="" id="image" alt="">
```

Comment:
Usually its good convention to add an alt e.g.
```html
<img src="" id="image" alt="My Funny Image">
```

https://www.w3schools.com/TAGS/att_img_alt.asp

Reply:
Great